source/rst/tax_smoothing_1.rst

```diff
@@ -1,18 +1,27 @@
+.. _tax_smoothing_1:
 
-How to pay for a war: part 1
-============================
+.. include:: /_static/includes/lecture_howto_py.raw
 
-An application of Markov jump linear quadratic dynamic programming
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. index::
+    single: python
+    
+    
+****************************
+How to Pay for a War: Part 1
+****************************
+
+.. contents:: :depth: 2
+
+**Co-author: Sebastian Graves**
```

Comment:
Maybe we could leave the link in to Seb's github as in the original? https://github.com/sebgraves